ssl/record/rec_layer_s3.c

```diff
@@ -869,27 +871,31 @@ int do_ssl3_write(SSL *s, int type, const unsigned char *buf,
             SSL3_RECORD_add_length(thiswr, 1);
 
             /* Add TLS1.3 padding */
+            rlen = SSL3_RECORD_get_length(thiswr);
+            max_padding = SSL3_RT_MAX_PLAIN_LENGTH - rlen;
```

Comment:
I was wondering if we'd ever allow a record length to be greater than SSL3_RT_MAX_PLAIN_LENGTH?